src/overlays/actors/ovl_Bg_Lotus/z_bg_lotus.c

```diff
@@ -20,22 +23,147 @@ const ActorInit Bg_Lotus_InitVars = {
     (ActorFunc)BgLotus_Init,
     (ActorFunc)BgLotus_Destroy,
     (ActorFunc)BgLotus_Update,
-    (ActorFunc)BgLotus_Draw
+    (ActorFunc)BgLotus_Draw,
 };
-*/
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_Bg_Lotus/BgLotus_Init.s")
+s32 D_80AD6D10[] = { 0x48580064 };
+
+void BgLotus_Init(Actor* thisx, PlayState* play) {
+    BgLotus* this = THIS;
+
+    s32 pad;
```

Comment:
```suggestion
    BgLotus* this = THIS;
    s32 pad;
```
No need for the extra newline